tests/functional/b/bad_open_mode.txt

```diff
@@ -1,6 +1,6 @@
-bad-open-mode:12:0::"""rwx"" is not a valid mode for open."
-bad-open-mode:13:0::"""rr"" is not a valid mode for open."
-bad-open-mode:14:0::"""+"" is not a valid mode for open."
-bad-open-mode:15:0::"""xw"" is not a valid mode for open."
-bad-open-mode:21:0::"""Ua"" is not a valid mode for open."
-bad-open-mode:22:0::"""Ur++"" is not a valid mode for open."
+bad-open-mode:12:0::"""rwx"" is not a valid mode for open.":HIGH
```

Comment:
Apparently it was already applied in some place. It's not important.

Reply:
Should I still apply the other commit?
And do we want to update the upgrader before this PR?

Reply:
If you know how to easily fix automatically with the upgrader we could do that but let's not do it manually :)